prime-router/docs/getting_started.md

```diff
@@ -307,6 +306,7 @@ If you would like to start with a fresh Vault database, you can clear the Vault
 cd prime_router
 docker-compose down -v
 rm -rf .vault/env/{key,.env.local}
+docker volume rm prime-router_vault # removes vault volume
```

Comment:
I just found out that docker-compose down -v does the remove of the volumes, so no need for this extra command.  Might as well just close the PR.

Reply:
Suggestion to close seconded; plus there's `cleanslate.sh` which prunes docker volumes as well...

Reply:
closing the PR based on above comments